libs/utils/utils.js

```diff
@@ -647,10 +647,12 @@ export function loadPrivacy() {
   };
   loadScript('https://www.adobe.com/etc.clientlibs/globalnav/clientlibs/base/privacy-standalone.js');
 
-  const privacyTrigger = document.querySelector('footer a[href*="#openPrivacy"]');
-  privacyTrigger?.addEventListener('click', (event) => {
-    event.preventDefault();
-    window.adobePrivacy?.showPreferenceCenter();
+  // Privacy triggers can exist anywhere on the page and can be added at any time
+  document.addEventListener('click', (event) => {
```

Comment:
> can be added at any time

Does this mean after the page has loaded?  This seems heavy handed to intercept all clicks and look for the link, but yeah if links get added after load then i guess it's the only solution, or possibly use a mutation observer to look for when it gets added and then add the click handler?  (that may not be a better soln)